### PR TITLE
DX-1 - Add typescript declaration notice to the shopware object

### PR DESCRIPTION
**Pull request by klarstil**
In this PR I'm adding a new notice to the shopware object article which points out we're shipping TypeScript declarations within the Shopware administration. They're helping internal as well as external developers to use the `Shopware` object. In the provided example GIF I'm showing how to register a new module and letting the IntelliSense help me to provide the necessary arguments and properties within the arguments.

### CODE REVIEW

**Comment by elkmod** [comment]
Nice example 👍  Could you give a quick heads up when this can be merged and add a little hint on the version/availability constraints?

File: guides/plugins/plugins/administration/the-shopware-object.md [hunk 1]
```diff
@@ -47,25 +47,32 @@ Learn more about them here: [Creating administration module](add-custom-module.m
...
+| Mixin      | A registry for `mixins`                                                                     |
+| Module     | A registry for `modules`                                                                    |
+| Plugin     | An interface to add `promise`based hooks to run when the administration launches            |
+| Service    | A helper to get quick access to service, e.g. `Shopware.Service('snippetService')`          |
+| Shortcut   | A registry for keyboard shortcuts                                                           |
+| State      | A wrapper for the [VueX](https://vuex.vuejs.org/) store to manage state                     |
+| Utils      | A collection of utility methods like `createId`                                             |
+
+## TypeScript declarations
+
```
Comment (elkmod) [inline guides/plugins/plugins/administration/the-shopware-object.md:69]: Add a hint on when this is available (SW version)

```
{% hint style="info" %}
TypeScript declarations are available from Shopware Version 6.4.? onwards.
{% endhint %}
```

**Review by elkmod — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/administration/the-shopware-object.md (hunk 1 above)
Comment (klarstil) [reply guides/plugins/plugins/administration/the-shopware-object.md:69]: Done, thank your for your support. It'll be part of the next minor release therefore 6.4.4.0

**Comment by klarstil** [comment]
@elkmod I'll give you a ping when platform MR got merged. Thanks for your support and your feedback, appreciate it 👍

**Review by PaddyS — APPROVED** [review]

**Comment by klarstil** [comment]
Ping @elkmod the corresponding MR for the administration got merged.

**Pull request merged**